build.ps1

```diff
@@ -24,11 +24,19 @@ if ( ! ( Test-Path $NuGetExecutable ) ) {
     Invoke-WebRequest $NuGetUrl -OutFile "$NuGetExecutable"
 }
 
+if ( ! ( Test-Path .nuget ) )
+{
+    Write-Output "Creating .nuget directory"
+    New-Item -ItemType Directory .nuget > $nul
+}
+
 if ( ! ( Test-Path .nuget\NuGet.exe ) ) {
+    Write-Output "Copying NuGet.exe into .nuget directory"
     Copy-Item $NuGetExecutable .nuget\NuGet.exe
 }
 
 # restore packages
+Write-Output "Restoring essential NuGet packages"

```

Comment:
These packages are essential for running the build script, the other packages are essential for building. 😉 

Perhaps "Restoring NuGet packages for build script"?